tests/unit_tests/test_config.py

```diff
@@ -0,0 +1,43 @@
+from collections.abc import Mapping
+import os
+
+import openmc
+import pytest
+
+
+@pytest.fixture(autouse=True, scope='module')
+def reset_config():
+    config = dict(openmc.config)
+    try:
+        yield
+    finally:
+        openmc.config.clear()
+        openmc.config.update(config)
+
+
+def test_config_basics():
+    assert isinstance(openmc.config, Mapping)
+    for key, value in openmc.config.items():
+        assert isinstance(key, str)
+        assert isinstance(value, os.PathLike)
```

Comment:
```suggestion
        assert isinstance(value, os.PathLike)
```
Could this be the type hint PathLike introduced recently https://github.com/openmc-dev/openmc/blob/aea56288198bb53ed753c6b3b48a7b767f4806ee/openmc/checkvalue.py#L9

Reply:
It would work, although the `_Config` class turns all its values into `Path` objects, so checking against `str` is not really necessary